Parsers/ASimNetworkSession/Parsers/vimNetworkSessionEmpty.yaml

```diff
@@ -81,8 +81,8 @@ ParserQuery: |
   , DstMacAddr:string // Optional
   , DstGeoCountry:string // Optional
   , DstGeoCity:string // Optional
-  , DstGeoLatitude:string // Optional
-  , DstGeoLongitude:string // Optional
+  , DstGeoLatitude:real // Optional
```

Comment:
Missing version update?

Reply:
It accidentally got here :-(. It is part of a different PR for network fixes. Anyways, I will update the version number as part of the network update PR.